translations/es/README.md

```diff
@@ -51,28 +53,25 @@ Puedes configurar:
 
 Este repositorio es una plantilla de GitHub, la cual contiene lo siguiente:
 
-- [example-notebook.ipynb](./example-notebook.ipynb), Este notebook utiliza la librería [Pandas](https://pandas.pydata.org/) para enseñar operaciones básicas con un pequeño archivo CSV (_Comma Separated Value_) [dataset](./wine-regions.csv)
-- [.devcontainer/Dockerfile](./.devcontainer/Dockerfile), para que pueda configurar qué sistema operativo utilizará el Codespace y cómo se debe construir el contenedor.
-- [.devcontainer/devcontainer.json](./.devcontainer/devcontainer.json), Un archivo de configuración utilizado por Codespaces para configurar [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), por ejemplo, para agregar y habilitar una extensión.
+- [example-notebook.ipynb](./example-notebook.ipynb): Un notebook que utiliza la librería [Pandas](https://pandas.pydata.org/) para enseñar operaciones básicas con un pequeño archivo CSV (_Comma Separated Value_) [dataset](./wine-regions.csv)
+- [.devcontainer/Dockerfile](./.devcontainer/Dockerfile): Archivo de configuración usado por Codespaces para determinar el sistema operativo y otros detalles.
+- [.devcontainer/devcontainer.json](./.devcontainer/devcontainer.json): Un archivo de configuración utilizado por Codespaces para configurar [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), por ejemplo, para agregar y habilitar extensiones adicionales.
 - `README.md`. Este archivo describe este repositorio y lo que contiene.
 
-### 🔎 ¿Ha encontrado un problema o tienes una idea para mejorarlo?
-Ayúdanos a mejorar este repositorio al [hacernos lo saber y abriendo un issue!](/../../issues/new). 
-
 ## 🧐 ¡Pruébalo!
 
 Prueba este repositorio de plantillas con Codespaces siguiendo estos pasos:
 
-1. Crea un repositorio desde esta plantilla. Utiliza este link [crea un repositorio](https://github.com/microsoft/codespaces-teaching-template-py/generate)
-1. Ve a la página principal del repositorio recién creado.
-1. Debajo del nombre del repositorio, usa el menú desplegable Código y, en la pestaña Codespaces, selecciona "Crear Codespace en main".
+1. Crea un repositorio desde esta plantilla. Utiliza este [link crea un repositorio](https://github.com/microsoft/codespaces-teaching-template-py/generate). Tú puedes hacer este repositorio privado o público, es tú decisión.
+2. Navega a la página principal del repositorio recién creado.
+3. Debajo del nombre del repositorio, usa el menú desplegable Código, y en la pestaña Codespaces, selecciona "Crear Codespace en main".
    ![Crea un codespace](https://docs.github.com/assets/cb-138303/images/help/codespaces/new-codespace-button.png)
-1. Se empieza a crear tu Codespace
+4. Espera mientras se inicializa tu Codespace
```

Comment:
This step change a bit in english. You can consider "Espera mientras GitHub inicializa el Codespace."